src/server/origin/address/controller.js

```diff
@@ -11,12 +11,12 @@ export const pageHeading =
  */
 export const originAddressGetController = {
   handler(req, h) {
-    const originAddress = req.yar.get('originAddress')
+    const { address } = req.yar.get('origin') ?? {}
 
     return h.view(indexView, {
       pageTitle,
       heading: pageHeading,
-      values: originAddress
+      values: address
     })
```

Comment:
I'm expecting to see a `nextPage` for this, pulled from the redirect_uri?

Worth putting in a corresponding test too, IMO.

Reply:
didnt put it here as this will also go to the summary page as is the last page in the flow - so whther it comes from summary or not, it will also go there, didnt see the point of the added complexity for no reason

Reply:
I can see the logic there - I suspect this'll get factored out a bit later.